lib/hanami/slice_registrar.rb

```diff
@@ -85,6 +82,16 @@ def with_nested
 
     private
 
+    def ancestors
```

Comment:
Could we please shift this method so it's below `find_slice_require_path`? It's only used by that one method and I generally find classes easier to follow when related methods like that stick together.

Reply:
Also, could you please put a little bit of a documentation comment on top of this one? It's a very abstract kind of method so it'd be useful to to have a one-liner explaining roughly what it does and why we use it.

(I know nothing else in this class has doc comments, but now's always a good time to start!)